Components/Flash/FlashTask.cpp

```diff
@@ -6,37 +6,30 @@
  ******************************************************************************
 */
 #include "FlashTask.hpp"
+#include <cstring>
+#include "Data.h"
 #include "GPIO.hpp"
-#include "SystemDefines.hpp"
-#include "Utils.hpp"
-#include "Timer.hpp"
 #include "RocketSM.hpp"
 #include "SPIFlash.hpp"
-#include "Data.h"
-#include <cstring>
+#include "SystemDefines.hpp"
+#include "Timer.hpp"
+#include "Utils.hpp"
 
 /**
  * @brief Constructor for FlashTask
  */
-FlashTask::FlashTask() : Task(FLASH_TASK_QUEUE_DEPTH_OBJS)
-{
-}
+FlashTask::FlashTask() : Task(FLASH_TASK_QUEUE_DEPTH_OBJS) {}
 
 /**
  * @brief Initialize the FlashTask
  */
-void FlashTask::InitTask()
-{
+void FlashTask::InitTask() {
     // Make sure the task is not already initialized
     SOAR_ASSERT(rtTaskHandle == nullptr, "Cannot initialize flash task twice");
-    
-    BaseType_t rtValue =
-        xTaskCreate((TaskFunction_t)FlashTask::RunTask,
-            (const char*)"FlashTask",
-            (uint16_t)FLASH_TASK_STACK_DEPTH_WORDS,
-            (void*)this,
-            (UBaseType_t)FLASH_TASK_RTOS_PRIORITY,
-            (TaskHandle_t*)&rtTaskHandle);
+
+    BaseType_t rtValue = xTaskCreate((TaskFunction_t)FlashTask::RunTask, (const char*)"FlashTask",
```

Comment:
should this have function call on same line and have the args be pushed to a different line? seems like its not consistent

Reply:
If the line is short enough it will make it one line. Fine imo.